app/controllers/ApplicationSubmissionController.scala

```diff
@@ -58,8 +59,11 @@ class ApplicationSubmissionController @Inject()(vatRegistrationService: VatRegis
           } else {
             applicantDetailsService.getApplicantDetails.map(_.contact.email)
           }
+          eligibilitySubmissionData <- vatRegistrationService.getEligibilitySubmissionData
```

Comment:
could probably just get the partyType and pass it to the view, letting it check whether it needs to render any extras based on it, a bit cleaner that way